man/git-reauthor.md

```diff
@@ -0,0 +1,83 @@
+git-reauthor(1) -- Rewrite history to change author's identity
+==============================================================
+
+## SYNOPSIS
+
+`git reauthor [<options>]`
+
+## DESCRIPTION
+
+Rewrite history to change author's identity.
+
+Goes through all existing commits and tags to selectively modify the identities:  
+- modify name and/or email  
+- affect author and/or committer identities  
+- specify the new name/email as parameters or use the ones from the Git config  
+- change all objects or only replace those from a specific identity (defined by its email)
+
```

Comment:
Describe how the utility works, rather than listing its features. You may have looked at some of our other man-pages, like the one for `git-effort`, but I think the description there is somewhat lacking too, so you should use man pages from git  as a guidelines instead. Unix utilites also have very good descriptions.

It's not easy, and require some thinking. Please ask if you get stuck, or push your suggestions and you will get a new review by someone.

Come to think of it, this is something we probably should mention in our upcoming Contributing Guidelines (#549).